lib/containers/runtime.pm

```diff
@@ -32,27 +34,73 @@ sub _rt_script_output {
     return script_output($self->runtime . " " . $cmd, @args);
 }
 
-=head2 build
+=head2 create_container($image, $name, [%args])
 
-Build a container.
+Creates a container.
+C<image> the name of the image we create the container from.
+C<name> is the given name of the created container.
+C<args> can be anything you want to run in the container, similar to C<run>. for 
+instance: C<docker create -it tumbleweed bash>
+
+=cut
+sub create_container {
+    my ($self, $image, $name, $args) = @_;
+    die 'wrong number of arguments' if @_ < 3;
+    $self->_rt_assert_script_run("container create --name $name $image $args", 300);
+    record_info "$name container created", "";
```

Comment:
Please use record_info as I suggested above. Sometimes, the first string doesn't fit in the frame if it's too long.